### ISSUE TITLE
Extremely weird mixin error that appears when using your mod

### ISSUE THREAD

**Issue by Sunconure11** [Ladysnake/Requiem#580]
https://pastebin.com/fGwb5Fzz

**Comment by Pyrofab** [comment]
This looks like another mod has a conflicting `@Overwrite` or `@Redirect` on the same method. Do you have an idea as to which mod is conflicting ?

**Comment by Sunconure11** [comment]
I’m not entirely sure.

**Comment by EmmyPynk** [comment]
Try removing Better Combat. I had this same issue so I took a look at your pastebin and noticed we both were using Better Combat. I removed Better Combat, now it loads perfectly.

**Comment by Sunconure11** [comment]
I'll have to do it later, I am busy with porting a mod to Quilt.

**Comment by Sunconure11** [comment]
@EmmyPynk This issue I'm having with porting a Quilt mod will probably take a few days to resolve. I suggest you talk with Pyro more in this issue.

**Comment by Sunconure11** [comment]
test

**Comment by Sunconure11** [comment]
I can get to this shortly. I've managed to get said mod on Quilt now.

**Comment by Sunconure11** [comment]
Not Simply Swords in my case. It is blaming Supplementaries. Will try without it.

https://pastebin.com/jeU37Eph

**Comment by Sunconure11** [comment]
Not supplementaries. I did get a crash report this time as well, but I don't know how much help it would be.

https://pastebin.com/U7pKLWDC

**Comment by Pyrofab** [comment]
Oh, maybe this is related to loading order. Does it come and go as you change the mod list ?

**Comment by Sunconure11** [comment]
It would seem this is probably the case

**Comment by Pyrofab** [comment]
Could you try again with beta 15 ? Shouldn't depend on load order now.

**Comment by Sunconure11** [comment]
https://gist.github.com/Sunconure11/beb999dfc5210623c0cd279d3c6fe65d

**Comment by Pyrofab** [comment]
Damn o_O

Does this happen with specific mods ?

**Comment by Sunconure11** [comment]
I think a mod is triggering it, yes, but I have no clue what.

**Comment by Sunconure11** [comment]
```
[18:40:51] [main/ERROR]: Mixin apply for mod requiem failed mixins.requiem.common.json:data.ItemPredicateMixin from mod requiem -> net.minecraft.class_2073: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException Unexpecteded NullPointerException whilst transforming the mixin class: [INJECT Applicator Phase -> mixins.requiem.common.json:data.ItemPredicateMixin from mod requiem -> Apply Injections -> PreInject -> mixins.requiem.common.json:data.ItemPredicateMixin from mod requiem->@Inject::fromJson(Lcom/google/gson/JsonElement;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lcom/google/gson/JsonObject;)V]
org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: Unexpecteded NullPointerException whilst transforming the mixin class: [INJECT Applicator Phase -> mixins.requiem.common.json:data.ItemPredicateMixin from mod requiem -> Apply Injections -> PreInject -> mixins.requiem.common.json:data.ItemPredicateMixin from mod requiem->@Inject::fromJson(Lcom/google/gson/JsonElement;Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfoReturnable;Lcom/google/gson/JsonObject;)V]
```

This looks to be of interest @Pyrofab

**Comment by Pyrofab** [comment]
Yeah, I figured it was of interest in some way, but it clearly does not happen with only requiem so it would be super helpful if I could know which circumstances are triggering it

**Comment by Sunconure11** [comment]
Hm, is there anything you can do without knowing what triggers it? I've been trying to debug stuff like this with other mods and have been unable to find what triggers such errors.

**Comment by Sunconure11** [comment]
Looking at the error some more, could you look into the ItemPredicateMixin and it’s relation to mods like Better Nether and other mods by the same devs?

**Comment by Sunconure11** [comment]
I have a good chance of losing power and/or internet due to the hurricane

**Comment by Sunconure11** [comment]
@Pyrofab check discord

**Comment by EightBlade** [comment]
I'm having a more or less identical issue.

**Comment by Redfan2** [comment]
Is this Issue still active? Otherwise it would be nice if you could close it. Thanks for taking the time to report it

**Comment by Sunconure11** [comment]
Haven't used Requiem, or rather, tried to, in a while, so I cannot say.

**Comment by Sunconure11** [comment]
@Redfan2 I think I know what causes the issue. Try looking into the following mods: Simply Swords, Knaves Needs, Better Combat, and anything that relies on them.

Edit: YEP IT IS THEM

**Comment by Sunconure11** [comment]
https://pastebin.com/iLeCaNK3
